exercises/isbn-verifier/description.md

```diff
@@ -32,4 +32,4 @@ It's getting even trickier since the check-digit of an ISBN-10 can be 'X'.
 
 * Generate a valid ISBN-13 from the input ISBN-10 (and maybe verify it again with a derived verifier)
 
-* Generate valid ISBN, maybe even from a given starting ISBN
\ No newline at end of file
+* Generate valid ISBN, maybe even from a given starting ISBN
```

Comment:
I wouldn't say adding newline is considered fixing typo

Reply:
Right reverted.